src/components/productDetails/productDetailsStyledComponent.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+const ProductDetailsWrapper = styled.div`
+  min-height: 90vh;
+  width: 100%;
+  padding: 20px;
+`;
+const ProductSubText = styled.p`
+  font-weight: 300;
+  font-size: 1em;
+`;
+
+const ProductDescriptionWrapper = styled.div`
+    padding: 40px;
+    padding-right:25%;
+`
+const TaxText = styled.p`
+    color: green;
+    font-size: 0.9em;
```

Comment:
we should not change font-size at multiple places. use a common typography across